timeless/roles/models.py

```diff
@@ -9,22 +19,25 @@ class Role(DB.Model):
     __tablename__ = "roles"
 
     id = DB.Column(DB.Integer, primary_key=True, autoincrement=True)
+    """
+        @todo #397:30m After adding role type, name is not useful and may make confusion.
+         It should be deleted and all tests or code using it should use type instead.
+    """
```

Comment:
@ammaratef45 Shouldn't we leave a puzzle to also alter the views for the Roles? For instance, the create/edit formulars, which probably still have the names? 

Or maybe not another puzzle, but at least give more details in this one so people know where to look.